default/generated/spring-framework/spring-framework-5.x-to-6.0-data-access.yaml

```diff
@@ -28,3 +28,36 @@
     url: https://github.com/hibernate/hibernate-orm/blob/5.6/migration-guide.adoc
   - title: 'Spring 6.0 migration guide'
     url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#removed-apis
+
+- ruleID: spring-framework-5.x-to-6.0-data-access-00010
+  category: mandatory
+  effort: 3
+  labels:
+    - konveyor.io/source=spring5
+    - konveyor.io/target=spring6+
+  when:
+    java.dependency:
+      name: org.hibernate.validator.hibernate-validator
+      upperbound: 6.2.5.Final
+  description: Spring 6 must use at least Hibernate Validator 7.0.x
+  message: |
+    Spring 6 must use at least Hibernate Validator 7.0.x. Consider, though, migrating directly to Hibernate Validator 8.0
```

Comment:
Let's go all the way to Hibernate Validator 8.x to keep rules as current as possible. No need to suggest older versions that might require upgrades sooner rather than later.